net/net.sh

```diff
@@ -779,12 +779,12 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --faucet-lamports ]]; then
       genesisOptions="$genesisOptions $1 $2"
       shift 2
-    elif [[ $1 = --operating-mode ]]; then
+    elif [[ $1 = --cluster-type ]]; then
```

Comment:
I went aggressive here not to retain compatibility, assuming these scripts isn't used for our any live clusters.